internal/cmd/rem.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/keygen-sh/keygen-relay/internal/licenses"
+	"github.com/spf13/cobra"
+)
+
+func RemCmd(manager licenses.Manager) *cobra.Command {
+	var id string
+
+	cmd := &cobra.Command{
+		Use:   "rem",
+		Short: "Delete a license from the local relay server's pool",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := manager.RemoveLicense(cmd.Context(), id)
+			if err != nil {
+				fmt.Println("error creating license record", err)
```

Comment:
Let's make sure that we're correctly writing to stderr for any errors.

Reply:
thank you!
I have fixed it!